python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/auto_materialize_policy_scenarios.py

```diff
@@ -136,6 +138,28 @@ def with_auto_materialize_policy(
                 PartitionKeyRange(start="2013-01-05-04:00", end="2013-01-07-03:00")
             )
         ],
+        expected_conditions={
+            **{
+                ("hourly", p): {MissingAutoMaterializeCondition()}
+                for p in hourly_partitions_def.get_partition_keys_in_range(
+                    PartitionKeyRange(start="2013-01-05-04:00", end="2013-01-07-03:00")
+                )
+            },
+            ("daily", "2013-01-05"): {
+                ParentOutdatedAutoMaterializeCondition(
+                    parent_asset_key=AssetKey("hourly"),
+                    parent_will_materialize=True,
```

Comment:
There's a strange non deterministic error here, where sometimes parent_will_materialize is True and sometimes False for this partition https://www.diffchecker.com/ENWRhbex/

Haven't been able to figure it out yet